disnake/abc.py

```diff
@@ -630,6 +630,23 @@ def jump_url(self) -> str:
         """
         return f"https://discord.com/channels/{self.guild.id}/{self.id}"
 
+    def _apply_implict_permissions(self, base: Permissions) -> Permissions:
```

Comment:
if `base` is being mutated in this method anyway, we don't need to return it again

Reply:
https://github.com/DisnakeDev/disnake/pull/1047/commits/136f4b16247e91a59ebbcd7e4696e9546345083b